ci_framework/roles/ci_network/defaults/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+cifmw_network_pre_cleanup: true
+cifmw_network_nm_config: []
+cifmw_network_nm_config_file: "/etc/NetworkManager/NetworkManager.conf"
+cifmw_network_generated_layout: "/etc/ci/env/network-layout.yml"
+# This parameter is provided as an exemple of the expected content.
```

Comment:
exemple -> example

Reply:
I'll make a follow-up, CI is at last green.